pkg/rancher-desktop/config/settings.ts

```diff
@@ -224,13 +224,24 @@ export async function clear() {
   await fs.promises.rm(paths.config, { recursive: true, force: true } as any);
 }
 
+/**
+ * Used for unit testing only.
+ * Could be used in core code if we ever want to reload changed deployment profiles, but that isn't needed now.
+ */
+export function clearSettings() {
+  settings = undefined;
+}
+
 /**
  * Load the settings file or create it if not present.  If the settings have
  * already been loaded, return it without re-loading from disk.
  */
 export function load(deploymentProfiles: DeploymentProfileType): Settings {
+  if (settings) {
+    return settings;
```

Comment:
A comment here that this explicitly doesn't touch the deployment profiles would be nice.

Reply:
Issue #5022 calls for refactoring, so such a comment won't be needed for long